common/test/acceptance/tests/studio/test_studio_home.py

```diff
@@ -33,7 +33,7 @@ def setUp(self):
         self.auth_page = AutoAuthPage(self.browser, staff=True)
         self.dashboard_page = DashboardPage(self.browser)
 
-    @flaky  # TODO: SOL-430
+    @flaky(max_runs=30, min_passes=30)
```

Comment:
It seems like this is clearly not flaky anymore. :) Please remove the flaky decorator before merging.